Universal Links vs Custom URL Schemes Support (#530)
italia/eidas-it-wallet-docs: docs/en/wallet-attestation.rst

      - JSON object with name/value pairs, identifying a Credential format supported by the Wallet.
      -
    * - **authorization_endpoint**

>       - URL of the Wallet Authorization Endpoint, preferably using a universal link for enhanced security and fallback support, though a custom url-scheme can also be used if necessary.

Reviewer: ```suggestion
      - URL of the Wallet Authorization Endpoint, it can be a universal link or a custom url-scheme.
```

we do not repeat the same words already wrote few lines above.

      - JSON String asserting the authentication level of the Wallet and the key as asserted in the cnf claim.
      -
    * - **authorization_endpoint**

>       - URL of the Wallet Authorization Endpoint, preferably using a universal link for enhanced security and fallback support, though a custom url-scheme can also be used if necessary.

Reviewer: ```suggestion
      - URL of the Wallet Authorization Endpoint, it can be a universal link or a custom url-scheme.
```
